CONTRIBUTING.md

```diff
@@ -34,8 +34,9 @@ Each commit and pull request should follow the [style guide][style_guide] and be
 
 Handy commands (run from the root directory):
 
+- `cargo tidy` runs tidy-checks (not including fmt/clippy)
```

Comment:
Question: What's the use case for a developer running `cargo tidy`?

Reply:
because usually devs will have run the other checks (build/fmt) but often forget to run the license check/etc

I'm also okay with a tidy task that is fmt+license+readme

Reply:
I think adding fmt would be most useful here.  I feel like the most common error is fmt, followed by license.

Reply:
Done